Component/Button/button.css

```diff
@@ -0,0 +1,198 @@
+@import url("../../Utility CSS/navbar.css");
+.box {
+  height: 14rem;
+  width: 120rem;
+  justify-content: flex-start;
+  padding: 2rem;
+}
+
+.btn {
+  height: 5rem;
+  width: 10rem;
+  font-size: 1.8rem;
+  margin: 2rem;
+  padding: 4px 6px;
+  border: none;
+  border-radius: 1rem;
+  color:var(--btn-primary-color
```

Comment:
The bracket for var seems to not have been closed - is the button color applying correctly?

Reply:
done